File: platform/sdk/nodejs/examples/cloud/cloud-003_list_buckets.js
const { CloudClient } = require("../../xuanji-sdk-cloud");
const client = new CloudClient({ region: "cn-east-1" });
client.createBucket("bucket-a");
client.createBucket("bucket-b");
const result = client.listBuckets();
if (!result.ok) process.exit(1);
console.log("XJ-OK: cloud-003_list_buckets");
process.exit(0);

File: platform/sdk/python/examples/cloud/cloud-003_list_buckets.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from xuanji_sdk_cloud import CloudClient

client = CloudClient({"region": "cn-east-1"})
client.createBucket("bucket-a")
client.createBucket("bucket-b")
result = client.listBuckets()
if not result["ok"]:
    sys.exit(1)
print("XJ-OK: cloud-003_list_buckets")
sys.exit(0)
